### DISCUSSION TITLE
descriptor type choices

### DISCUSSION THREAD

**Discussion by Feiyang472** [deepmodeling/deepmd-kit#164] [Q&A] [answered]
In the training file what are possible choices for `model->descriptors->type` other than `"loc_frame"`?

Sorry if this has been brought up. I just couldn't find it in any of the documentation.

**Comment by njzjz** [comment, marked as answer]
https://github.com/deepmodeling/deepmd-kit/blob/984ef69fbf9cecae80de91ac7c2126e0e22398e5/source/train/Trainer.py#L76-L86

Four types are supported: `loc_frame`, `se_a`, `se_r`, and `se_ar`. The last 3 types are based on [the DeePot-SE method](http://papers.nips.cc/paper/7696-end-to-end-symmetry-preserving-inter-atomic-potential-energy-model-for-finite-and-extended-systems).

**Comment by Feiyang472** [comment]
Thank you for your reply.
